CHANGELOG.md

```diff
@@ -29,6 +29,8 @@
 - Ability for UDL to use external trait interfaces [#1831](https://github.com/mozilla/uniffi-rs/issues/1831)
 - Add support for docstrings via procmacros [#1862](https://github.com/mozilla/uniffi-rs/pull/1862)
 - Objects can now be returned from functions/constructors/methods without wrapping them in an `Arc<>`.
+- Proc-macro function/method arguments can now have defaults
+- Proc-macros defaults now support an empty vecs and the inner value of an `Option<T>`
```

Comment:
nit - s/an empty/empty/ - and do we need 2 lines here?

Also, "and the inner value of an `Option<T>`" seems stale? ie, the `Some()` part is now needed.